pom.xml

```diff
@@ -590,5 +566,45 @@
                 </dependency>
             </dependencies>
         </profile>
+        <profile>
+         <id>travis</id>
+         <activation>
+            <property>
+                <name>requireMavenVersion</name>
+                <value>3.5.2</value>
+            </property>
+         </activation>
+          <build>
+          <plugins>
+		  <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-enforcer-plugin</artifactId>
+				<version>3.0.0-M2</version>
+				<executions>
+					<execution>
+						<id>enforce-maven</id>
+						<goals>
+							<goal>enforce</goal>
+						</goals>
+						<configuration>
+							<rules>
+								<requireMavenVersion>
+									<version>3.5.4</version>
+								</requireMavenVersion>
+								<requireJavaVersion>
+									<version>11</version>
+									<message>
+										HAPI FHIR is still targeting JDK 8 for published binaries, but we require JDK 11
+										to build and test this library.
+									</message>
+								</requireJavaVersion>
+							</rules>
+						</configuration>
+					</execution>
+				</executions>
+          </plugin>
+		  </plugins>          
+          </build>
```

Comment:
Thanks @gitcliff , in a profile , you dont need to repeat all these other elements,
i think you only need  `<groupId>org.apache.maven.plugins</groupId>`  `<artifactId>maven-enforcer-plugin</artifactId>`
`<version>3.0.0-M2</version>`  and `<requireMavenVersion>`  .
please dont re-add there other elements.

Reply:
@mozzy11 adding only those elements wasn't working so i added the other elements to work